plusplusbot/helpers.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from unidecode import unidecode
+
+
+class ScottFactorExceededException(Exception):
+    pass
+
+
+def sanitize_emojirade(text):
+    # Lowercase the text
+    lowered = text.lower()
+
+    # Strip whitespace
+    stripped = lowered.strip()
+
+    # Remove any random misc chars we deem unnessesary
+    scrubbed = re.sub("['\"-_+=]", "", stripped)
+
+    # unidecode will normalize to ASCII
+    normalized = unidecode(scrubbed)
+
+    return normalized
+
+
+def match_emojirade(guess, emojirades, scott_factor=2):
+    guess = re.escape(guess)
+
+    for emojirade in emojirades:
+        if not (len(emojirade) / scott_factor) < len(guess) < (len(emojirade) * scott_factor):
```

Comment:
does the guess need to be more than lower scott_factor? not sure how to describe the lower and higher limit. 
Anyway, if they are shorter, it won't match anyway?

Reply:
Ahh good point, if it's smaller, the it'll never match, the goal here was to stop too big of a guess being posted! Will remove the lower bound check.